modules/index/src/main/java/org/apache/ignite/internal/index/IndexBuildController.java

```diff
@@ -250,8 +277,8 @@ private void stopBuildingIndexesIfPrimaryExpired(TablePartitionId replicaId) {
         }
     }
 
-    private CompletableFuture<MvTableStorage> getMvTableStorageFuture(long causalityToken, TablePartitionId replicaId) {
-        return indexManager.getMvTableStorage(causalityToken, replicaId.tableId());
+    private CompletableFuture<MvTableStorage> getMvTableStorageFuture(long causalityToken, int tableId) {
```

Comment:
Why did you change it to `int tableId`, before that it wasn’t bad.

Reply:
because we need only tableId for `org.apache.ignite.internal.index.IndexManager#getMvTableStorage`, and semantically we use table id, not replica id, to get MvTableStorage. I don't see a reason to pass `TablePartitionId replicaId` to `getMvTableStorageFuture` then